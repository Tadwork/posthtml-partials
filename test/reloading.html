<mixin name="say">
	<p>Hello!</p>
</mixin>

<mixin name="say" from>
	<p>Hello from {{ from }}!</p>
</mixin>

<mixin name="say" from with>
	<p>Hello from {{ from }} with {{ with }}!</p>
</mixin>

<div>
	<mixin name="say"></mixin>
</div>

<div>
	<mixin name="say" from="mixin" with="love"></mixin>
</div>

<div>
	<mixin name="say" from="mixin"></mixin>
</div>
