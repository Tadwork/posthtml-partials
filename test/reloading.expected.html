





<div>
	
	<p>Hello!</p>

</div>

<div>
	
	<p>Hello from mixin with love!</p>

</div>

<div>
	
	<p>Hello from mixin!</p>

</div>
